pkg/services/rendering/interface.go

```diff
@@ -12,6 +12,7 @@ import (
 var ErrTimeout = errors.New("timeout error - you can set timeout in seconds with &timeout url parameter")
 var ErrConcurrentLimitReached = errors.New("rendering concurrent limit reached")
 var ErrRenderUnavailable = errors.New("rendering plugin not available")
+var ErrServerTimeout = errors.New("error trying to connect to image-renderer service")
```

Comment:
I wonder if this should better become an [errutil](https://github.com/grafana/grafana/blob/main/contribute/architecture/backend/errors.md#declaring-errors) one

Reply:
In that case it isn't an error that we are using as response of a request. I mean, we don't need to set any status code. 

Are we moving all errors to `errutil`? So in that case, should be put `StatusUnknown` status?

Reply:
For conformance and also because we don't always know whether a service will expose an API in the future I'm in favour of all services to switch return `errutil` errors.
No opinion regarding the returning status code, especially if the specific one is not used in an API response; `StatusUnknown` or `StatusInternal` seem fine.

Reply:
Done